packages/config/eslintrc.base.cjs

```diff
@@ -13,5 +13,13 @@ module.exports = {
   env: {
     es6: true,
   },
-  plugins: ['@typescript-eslint'],
+  plugins: ['@typescript-eslint', 'unused-imports'],
+  rules: {
+    'no-unused-vars': 'off', // We use the unused-imports plugin instead
+    'unused-imports/no-unused-imports': 'error',
+    'unused-imports/no-unused-vars': [
+      'warn',
+      { vars: 'all', varsIgnorePattern: '^_', args: 'after-used', argsIgnorePattern: '^_' },
```

Comment:
Please remove the ignore pattern for `_`.

Reply:
Done. Removed `IgnorePattern` for var & args starting with `_`